relation_engine_server/utils/wait_for.py

```diff
@@ -0,0 +1,89 @@
+"""
+Block until all dependent services come online.
+"""
+import requests
+import time
+import sys
+from .config import get_config
```

Comment:
Let's use the full import paths, so this would be `from relation_engine_server.utils.config import get_config`